spark-etl/src/main/scala/geotrellis/spark/etl/config/Input.scala

```diff
@@ -0,0 +1,13 @@
+package geotrellis.spark.etl.config
+
+import org.apache.spark.storage.StorageLevel
+
+case class Input(
+  name: String,
+  format: String,
+  backend: Backend,
+  cache: Option[StorageLevel] = None,
+  noData: Option[Double] = None
+) {
+  def params = getParams(backend.`type`, backend.path)
```

Comment:
This branches on backend type. Should `Backend` have a type hierarchy? In that case this could be more clearly explained as overloading an abstract method.